docs/user_guide/model_execution.md

```diff
@@ -0,0 +1,228 @@
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Concurrent Model Execution
+
+The Triton architecture allows multiple models and/or multiple
+instances of the same model to execute in parallel on the same
+system. The system may have zero, one, or many GPUs. The following
+figure shows an example with two models; model0 and model1. Assuming
+Triton is not currently processing any request, when two requests
+arrive simultaneously, one for each model, Triton immediately
+schedules both of them onto the GPU and the GPU’s hardware scheduler
+begins working on both computations in parallel. Models executing on
+the system's CPU are handled similarly by Triton except that the
+scheduling of the CPU threads execution each model is handled by the
+system's OS.
+
+![Triton Mult-Model Execution Diagram](images/multi_model_exec.png)
+
+By default, if multiple requests for the same model arrive at the same
+time, Triton will serialize their execution by scheduling only one at
+a time on the GPU, as shown in the following figure.
+
+![Triton Mult-Model Serial Execution
+Diagram](images/multi_model_serial_exec.png)
+
+Triton provides a [model configuration option called
+instance-group](model_configuration.md#instance-groups) that allows
+each model to specify how many parallel executions of that model
+should be allowed. Each such enabled parallel execution is referred to
+as an *instance*. By default, Triton gives each model a single
+instance for each available GPU in the system. By
+using the instance_group field in the model configuration, the number
+of execution instances for a model can
+be changed. The following figure shows model execution when model1
+is configured to allow three instances. As shown in the figure, the
+first three model1 inference requests are immediately executed in
+parallel. The fourth model1 inference request must wait until one of
+the first three executions completes before beginning.
+
+![Triton Mult-Model Parallel Execution
+Diagram](images/multi_model_parallel_exec.png)
+
+# Models And Schedulers
```

Comment:
This Model Execution doc feels a bit strange. The concurrent execution section at the top makes sense, but the following sections on schedulers, stateless, control inputs, etc. feel like they don't belong in this category/doc.